ceph.go

```diff
@@ -342,8 +379,11 @@ func pgDumpPgsBrief() []*pgBriefItem {
 	pgBriefs = sanitizePgBriefs(pgBriefs)
 
 	puis := pgUpmapItemMap()
+	pools := osdPoolDetails()
 	for _, pgb := range pgBriefs {
-		reorderUpToMatchActing(puis[pgb.PgID], pgb.Up, pgb.Acting)
+		if !pools.PgUsesEC(pgb.PgID) {
```

Comment:
IMO this should just go into `reorderUpToMatchActing` so that we don't have to put the guard everywhere. Feel free to rename the function as necessary.

Reply:
I see, I've now restructured the function to do that